core/docker/compose.sh

```diff
@@ -1,6 +1,11 @@
 composeFilePath=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 
 if [ "$1" == "up" ]; then
+    docker-compose -f "$composeFilePath"/docker-compose-mongo.yml up -d
+
+    # Set up the replica set
+    "$composeFilePath"/initiateReplicaSet.sh
```

Comment:
@BMartinos @rcrichton could we add another startup option? `init`. This option could contain the Importer script and the Mongo init scripts. 

Currently, if we were to stop the Instant services and then try to use `up` to start them again there will be problems - this is an existing issue as the importer script would have silently failed on the re-use of `up`. But now we are going to re-initialize the Mongo replica set. This shouldn't fail - but it is unnecessary.

Reply:
@MattyJ007 how would the `init` option work in practice? Normal compose first and then init? There might be some dependencies making this a bit harder to resolve. For replica set to work we need to mongos running, For the importer to work we need the core stored, which needs mongo to work properly

Reply:
Init would basically replace `up` and up would then just be `docker-compose up -f ...` (no importer script or replica specific stuff)

Reply:
Ok cool i see. So its a duplicate of the `up` option which contains all the scaffolding related stuff. And then the `up` option will just do the startup and no configuration setup. I think that sounds fine to me :+1:

Reply:
addressed in https://github.com/openhie/instant/pull/41/commits/26046074e16c00c733162602fec1a77fc888941b